tests/console/nftables.pm

```diff
@@ -23,16 +23,23 @@ sub run {
 
     # check nftables, stop iptables and start firewalld by need
     my $logs = "/tmp/log";
+    my $dir  = "/etc/nftables";
 
-    #check that nftables has been already installed
+    # check that nftables has been already installed
     assert_script_run('echo -e "____ nftables version and rules ____\n\n" > ' . $logs);
-    assert_script_run("rpm -qa nftables |& tee -a $logs");
+    assert_script_run("zypper info nftables |& tee -a $logs");
```

Comment:
What's wrong about `rpm -qa`? `rpm -qa` can be much faster than zypper because it involves no remote operation

Reply:
because from how it's being called here, the call will be hidden in a log file. the remote operation takes not a lot of time and gives much more insight than the simple call to rpm.

Reply:
ok, what information is needed?

Reply:
The test does not install nftables on it's own, so if there's a newer version available it would be hidden, same as the repo where it was installed from. 

Without running into a "what about", I consider that information more valuable rather than saving few seconds.

Reply:
ah, cool

Reply:
@Zaoliang also address this change.

Reply:
@Zaoliang https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12113#discussion_r597638425

Reply:
done